cves/CVE-2017-14056.yml

```diff
@@ -125,7 +131,12 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
+  answer: |
+    The code specific to the vulnerability was not changed much throughout
+    its existence. The lines specific to it where only changed once, where a function's
+    name was changed. The vulnerability did exist for over 9 years, so many small 
```

Comment:
Good analysis. I like how you explicitly stated the timeline, which makes this more interesting (9 years? wow that's scary).